posts/regular-expression-patterns.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: How to Search with Sourcegraph Using Regular Expression Patterns
```

Comment:
I used title capitalization based on [Chicago manual of style](https://capitalizemytitle.com/style/Chicago/). @ltagliaferri Do you have a different style guide to recommend for titles?

Reply:
The guidance I have used is that "How To" helps with SEO

Reply:
We will create an in-house style guide soon